core/src/main/resources/application.yml

```diff
@@ -51,6 +51,12 @@ feast:
     disableRestControllerAuth: false
 
   logging:
+    # Logging forwarder currently provides a machine readable structured JSON log to an
+    # external fluentd service that can give better insight into what is happening in Feast.
+    logForwarderProperties:
+      enabled: false
```

Comment:
What configuration is necessary in order to log using the forwarder but **NOT** log to the console?